library/Email.re

```diff
@@ -0,0 +1,38 @@
+open Cohttp_lwt_unix;
```

Comment:
top can you call this Sendgrid? as we're totally going to extract this to a package